<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeepArcTimeline</class>
 <widget class="qMRMLWidget" name="DeepArcTimeline">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>205</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="btn_collapsible_deeparctimeline">
     <property name="text">
      <string>DeepArc Timeline</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <property name="buttonTextAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indicatorAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Input Slice</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="slice_selector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLSliceNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input Sequence Browser</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="sequence_browser_selector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLSequenceBrowserNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_toggle_timeline">
          <property name="text">
           <string>Toggle Timeline</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_load">
          <property name="text">
           <string>Load Timeline</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QProgressBar" name="progress_bar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>DeepArcTimeline</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>sequence_browser_selector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeepArcTimeline</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>slice_selector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
